### ISSUE TITLE
fix handle missing state resource type

### ISSUE THREAD

**Issue by fatonramadani** [windmill-labs/windmill#2092]
[MESSAGE_ID=1141000549106470972]

**Comment by hmacr** [comment]
@fatonramadani I'm not sure if these issues are up for grabs; I'd love to contribute to the windmill backend if there are open issues (or) feats that need to be implemented. (just asking this query in a random open issue). Thanks!

**Comment by rubenfiszel** [comment]
Hey @hmacr , thanks for offering.

We need to improve our todo workflow to have enough context for contributors. We just recently started using a command in discord to create those todos.

Most of those small bugs take as much time to document properly for external contributors  than to fix ourselves.  The easiest would be to take on  medium sized features or even to come up with features you'd like to see in windmill. Is there anything in windmill you'd like to improve?

**Comment by hmacr** [comment]
@rubenfiszel Thanks for the info.

I haven't used windmill a lot; just been reading up the backend rust code. I'll create an issue when I think of something that could improve windmill. Just wondering if the medium-sized features are planned to be implemented by the team?

**Comment by rubenfiszel** [comment]
@hmacr yes we plan to implement everything but if a feature is identified by being tackled on by someone else, we would give him priority.
